wallet-tool.py

```diff
@@ -229,6 +230,35 @@ def cus_print(s):
         fd.write(walletfile)
         fd.close()
         print('saved to ' + walletname)
+elif method == 'changepassphrase':
+    """ Changes encryption password directly..
+    Replaces old encrypted seed with new seed. Users should have their seed
+    backed up
+    """
+    seed = wallet.seed # old seed
+    words = mn_encode(seed)
+    print("WARNING: Please remember to have a backup of the seed or old" +
+    " wallet.json before attempting to change the passphrase.")
+    password = getpass.getpass('Enter new wallet encryption passphrase: ')
+    password2 = getpass.getpass('Reenter new wallet encryption passphrase: ')
+    if password != password2:
+        print('ERROR. Passwords did not match')
+        sys.exit(0)
+    password_key = btc.bin_dbl_sha256(password)
+    encrypted_seed = encryptData(password_key, seed.decode('hex'))
+    timestamp = datetime.datetime.now().strftime("%Y/%m/%d %H:%M:%S")
+    walletname = args[0]
+    walletpath = os.path.join('wallets', walletname)
+    fd = open(walletpath, 'r')
+    walletfile = fd.read()
+    fd.close()
+    walletdata = json.loads(walletfile)
+    walletdata['creation_time'] = timestamp
```

Comment:
Yes, that wallet creation time one day needs to be used for #470 and https://github.com/JoinMarket-Org/joinmarket/issues/55#issuecomment-184165679